src/app/features/StepDefinitionEditor/Models/MockModel.js

```diff
@@ -40,7 +40,7 @@ var createMockModelConstructor = function (
         var ast = astCreatorService;
 
         var data = {
-            url: ast.literal(new RegExp(this.url))
+            url: ast.literal(new RegExp(this.url.replace(/\?/g,'\\\?')))
```

Comment:
This is cool, but regexs can be a bit cryptic, especially when escaping stuff. Maybe add some comments as to why we have to do this?